CMakeLists.txt

```diff
@@ -17,23 +17,20 @@ ELSE()
 ENDIF()
 
 # The source of the library we want to build
-add_library(${PROJECT_NAME} src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
+add_library(${PROJECT_NAME} SHARED src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
 
+#Add the VM dependency
 add_subdirectory(dependencies/pharo-vm)
+target_link_libraries(${PROJECT_NAME} PharoVMCore)
```

Comment:
This makes that building this project builds the vm automatically as a dependency (generating header files and others), thus not requiring to do it manually anymore

Reply:
@guillep fantastic. And easier than I've thought! Thanks a lot!

Reply:
Also notice that making this imports the correct headers for all platforms :)
So this now works for other than macOS!

Reply:
@guillep you rock!